vraptor-core/src/main/java/br/com/caelum/vraptor/serialization/gson/DateGsonConverter.java

```diff
@@ -61,5 +61,9 @@ public Date deserialize(JsonElement json, Type typeOfT, JsonDeserializationConte
 			throw new JsonSyntaxException(json.getAsString(), e);
 		}
 	}
+	
```

Comment:
Only subclasses will override this method, so the properly visibility is `protected`.